src/gov-action-contracts/AIPs/UpgradeArbOS/SetWASMModuleRootAction.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "@arbitrum/nitro-contracts/src/rollup/IRollupLogic.sol" as rl;
+import "@arbitrum/nitro-contracts/src/rollup/IRollupCore.sol" as rc;
```

Comment:
Why are we aliasing this? I think you don't need to import `IRollupCore.sol` explicitly as it is imported in `IRollupLogic.sol` already, or you can do this to avoid collision
```suggestion
import { IRollupLogic } from "@arbitrum/nitro-contracts/src/rollup/IRollupLogic.sol";
import { IRollupCore } from "@arbitrum/nitro-contracts/src/rollup/IRollupCore.sol";
```

Reply:
there's a naming collision w/ interfaces in in interfaces.sol

Reply:
I haven’t test but what I suggested might work?

Reply:
nah, the `IRollupCore` name is declared twice